media.py

```diff
@@ -26,11 +26,10 @@ def __init__(self, filename):
         self.data = self.get_data()
         self.get_poster()
 
-
+    @property
```

Comment:
Oh that's sweet. Is this convention for getters? if I have an instance variable of "foo" is it convention to keep that variable private and then add?



```python
class PropertyExample:

    def __init__(self):
        self.foo = 'foo bar string'

    @property
    def getFoo(self):
        return self.foo
```